BuildingSync.xsd

```diff
@@ -17787,4 +17790,468 @@
+                              <xs:element name="Beta1" type="xs:decimal" minOccurs="0" maxOccurs="1">
+                                <xs:annotation>
+                                  <xs:documentation>In a two and three parameter model, this is the slope of the line (Figure D-1 (a)-(d)).  If the model type is a 3p heating model, this is referred to as beta_hdd, whereas for a 3p cooling model, this is referred to as beta_cdd (per CalTRACK terminology).  In the 4p and 5p models, this is beta_hdd.</xs:documentation>
+                                </xs:annotation>
+                              </xs:element>
+                              <xs:element name="Beta2" type="xs:decimal" minOccurs="0" maxOccurs="1">
+                                <xs:annotation>
+                                  <xs:documentation>In both three parameter models, this is the change point.  In the 4p and 5p models, this is beta_cdd.</xs:documentation>
+                                </xs:annotation>
+                              </xs:element>
+                              <xs:element name="Beta3" type="xs:decimal" minOccurs="0" maxOccurs="1">
+                                <xs:annotation>
+                                  <xs:documentation>In the 4p models, this is the change point (as there is only one change point).  In the 5p models, this is the lower value change point, which in CalTRACK terms is referred to as the heating change point.</xs:documentation>
+                                </xs:annotation>
+                              </xs:element>
+                              <xs:element name="Beta4" type="xs:decimal" minOccurs="0" maxOccurs="1">
+                                <xs:annotation>
+                                  <xs:documentation>In the 5p model, this is the upper value change point, which in CalTRACK terms is referred to as the cooling change point.</xs:documentation>
+                                </xs:annotation>
+                              </xs:element>
+                            </xs:sequence>
+                          </xs:complexType>
+                        </xs:element>
+                        <xs:element name="TimeOfWeekTemperatureModel">
+                          <xs:complexType>
+                            <xs:sequence>
+                              <xs:element name="DUMMY"/>
+                            </xs:sequence>
+                          </xs:complexType>
+                        </xs:element>
+                      </xs:choice>
+                    </xs:complexType>
+                  </xs:element>
+                  <xs:element name="DerivedModelPerformance">
+                    <xs:annotation>
+                      <xs:documentation>Characterization of the performance of the model.</xs:documentation>
+                    </xs:annotation>
+                    <xs:complexType>
+                      <xs:sequence>
+                        <xs:element name="RSquared" minOccurs="0" maxOccurs="1">
```

Comment:
add annotation?

Reply:
`a measure of the extent to which variations in the dependent variable from its mean value are explained by the regression model` - ASHRAE Guideline 14-2014

Reply:
fixed